docs/topics/testing/tools.txt

```diff
@@ -880,6 +880,32 @@ It also provides an additional method:
     modify them, you could reload them in the ``setUp()`` method with
     :meth:`~django.db.models.Model.refresh_from_db`, for example.
 
+.. note::
+
+    Testing code which uses :func:`~django.db.transaction.on_commit` hooks are
+    not run during the execution of :class:`~django.test.TestCase` tests. As
+    mentioned above, the proper way to test this is to use the
+    :class:`~django.test.TransactionTestCase`. However, should it be required
+    to trigger :func:`~django.db.transaction.on_commit` in a
+    :class:`~django.test.TestCase` tests, one possible solution is to add
```

Comment:
either "in a TestCase test" or "in testCase tests", drop the `a` or the `s`

Reply:
Fixed 😅